scripts/eia/doi_Nigeria-ATAFI-Validations.R

```diff
@@ -18,20 +18,17 @@ carob_script <- function(path) {
    
    meta <- data.frame(
       # Need to fill-in metadata...
-      # carobiner::read_metadata(uri, path, group, major=2, minor=0),
-      uri = carobiner::simple_uri(uri),
-      dataset_id = uri,
       authors = "Mary Jane;John Do",
-      data_citation="...",
-      data_institute = NA ,
```

Comment:
This is `AfricaRice` from [values/values_institute.csv#L3](https://github.com/reagro/terminag/blob/483e21820604113ae7adb8a9a190ab4fcd125382/values/values_institute.csv#L3)